### PR TITLE
Jenkins Troubleshooting and Multiple Stores Warning

### PR DESCRIPTION
**Pull request by ThomasLehnerSpryker**
## PR Description

Added troubleshooting article for jenkins and warning on how to for creating multiple stores, indicating compatibility issues with paas

## Checklist
- [x ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+* Troubleshooting installation issues
+* Troubleshooting Spryker in Docker issues
+* Troubleshooting general technical issues
+
+Every troubleshooting article should have its own page in the respective section.-->
+
+## Title
+
+Jenkins does not restart properly due to long running job
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md:17]: Why does Jenkins need to restart during a deployment? How does Jenkins not being able to restart fails a deployment?

File: docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## {Description}
+
+You may encounter problems when trying to do new deployments on your environments if Jenkins is running a job that is usually running for an extended period of time and cannot be completed before the deployment. Typical symptoms of this problem is that deployment fails in Deploy_Scheduler deployment step.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md:21]: Are customers contacting us regarding failed deployments at `Deploy_Scheduler` step or Jenkins?

File: docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Cause
+
+The cause of this problem is that long running, heavy jobs might saturate the resources that Jenkins has available to a degree that the instance cannot follow the stop command it receives during deploy and the pipeline will subsequently fail at this step.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md:25]: By `instance` do you mean the Jenkins instance?

Comment (ThomasLehnerSpryker) [reply docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md:25]: yes

File: docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md (hunk 2 above)
Comment (ThomasLehnerSpryker) [reply docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md:21]: Both, the reports are varying

File: docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md (hunk 1 above)
Comment (ThomasLehnerSpryker) [reply docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md:17]: to apply changes made during deployment

**Review by andriitserkovnyi — APPROVED** [review]

**Pull request merged**